packages/fxa-content-server/app/styles/modules/_legal.scss

```diff
@@ -21,10 +21,6 @@
 
     a {
       padding: 0 16px;
-
-      &:last-child {
-        padding: 16px 0 16px 16px;
```

Comment:
It's unclear to me why this was styled like this to begin with, as it looks somewhat specific. That being said, I think it's fine to remove this (the vertical padding), but the horizontal padding pushes the item in 16px on the right. Just so we can maintain it flush against the edge can we also set `margin-right: -16px;` using this selector?

Reply:
Good catch, I did not notice the horizontal padding.

Reply:
Hi @byrw I think we can actually forego the left margin on the privacy link, this seems to add excess space between the two items; I was referring to using a negative margin on the privacy link/last-child to offset it's 16 padding. Doing this will ensure it's inline with the "Sign out" button at the top, and the white container on smaller screens.

Reply:
Sorry, I thought you mistyped.